### PR TITLE
chore(camunda-bpm-run): Add WADL Config Docs

### PR DESCRIPTION
**Pull request by psavidis**
**Related-to**: https://github.com/camunda/camunda-bpm-platform/issues/4508

### CODE REVIEW

**Comment by psavidis** [comment]
### Notes to the Reviewer

**Context**: Exposure of WADL File endpoint via the REST API
**What**: This PR adds [docs](https://github.com/camunda/camunda-bpm-platform/issues/4508#:~:text=Property%20can%20be%20added%20to%20the%20docs/user%20guide.) for the newly added property

File: content/user-guide/camunda-bpm-run.md [hunk 1]
```diff
@@ -286,6 +286,25 @@ If you want to allow cross-origin requests to the [REST API]({{< ref "/reference
   </tr>
 </table>
 
+## REST
+
+`Camunda-run` can be configured to disable the REST endpoint which exposes the WADL file via a property.
```
Comment (danielkelemen) [inline content/user-guide/camunda-bpm-run.md:291]: 🔧 `Camunda-run` looks a bit weird. I think usually we just refer to it as Camunda Run. Also not in code block:

```suggestion
Camunda Run can be configured to disable the REST endpoint which exposes the WADL file via a property.
```

File: content/user-guide/camunda-bpm-run.md [hunk 2]
```diff
@@ -286,6 +286,25 @@ If you want to allow cross-origin requests to the [REST API]({{< ref "/reference
...
+  <tr>
+      <th>Prefix</th>
+      <th>Property name</th>
+      <th>Description</th>
+      <th>Default value</th>
+  </tr>
+  <tr>
+      <td rowspan="15"><code>camunda.bpm.run.rest</code></td>
+      <td><code>.disable-wadl</code></td>
+      <td>Disables the REST endpoint `/application.wadl`</td>
```
Comment (danielkelemen) [inline content/user-guide/camunda-bpm-run.md:303]: 💬 Just thinking, maybe we could add a little text about what it actually is.

```suggestion
      <td>Disables the REST endpoint `/application.wadl`. Web Application Description Language (WADL) is an XML description of the deployed RESTful web application.</td>
```

**Review by danielkelemen — COMMENTED** [review]
👍 Looks good. I added two notes, let me know what you think.

**Review by danielkelemen — APPROVED** [review]

File: content/user-guide/camunda-bpm-run.md (hunk 2 above)
Comment (psavidis) [reply content/user-guide/camunda-bpm-run.md]: Sure, that would be fine by me. I've adjusted also the code characters notation which were not appearing correct.
This is the final result:

<details><summary>Screenshot</summary>
Screenshot 2024-09-16 at 5 12 42 PM
</details>

**Pull request merged**